docs/exodus-intro.md

```diff
@@ -23,16 +23,16 @@ It uses:
 * Parallelism - can run compilations and tests in parallel.
 * Hermeticity - builds can access only what is declared and pre-fetched, so they run 'hermetically' without unnecesary noise from additional files. 
 
-### Multi-language environment
```

Comment:
currently we migrated jvm languages, but we are thinking of migrating node.js and maybe even react as well